docs/_docs/extras/minimal-deploy-iam.md

```diff
@@ -169,6 +169,23 @@ Finally, create a user and add the user to IAM group. Here's an example:
     aws iam create-user --user-name tung
     aws iam add-user-to-group --user-name tung --group-name Jets
 
+## Alternately create policy, group, and user in AWS console
+
+The above steps can alternately be created in AWS Console.
+
+Go to [IAM Policies](https://console.aws.amazon.com/iam/home?nc2=h_m_sc#/policies).
+Click "Create policy", then "JSON", then "Next: tags", then "Next: review".
+Name the policy "JetsPolicy" and click "Create policy".
+
+Go to [IAM Groups](https://console.aws.amazon.com/iam/home?nc2=h_m_sc#/groups).
+Click "Create new group". Name the group "Jets" and click "Next step".
+Search for "JetsPolicy", check its checkbox, click "Next step", then "Create group".
+
+Go to [IAM Users](https://console.aws.amazon.com/iam/home?nc2=h_m_sc#/users).
+Click "Add user". Give the user a name and check "Programmatic access".
+Click "Next: permissions". Check the "Jets" group to add user to group.
+Click "Next: tags", "Next: Review", then "Create user".
+
```

Comment:
I wasn't sure if this would be an extra benefit but ran through it and documented just in case.

Is it typical that users would have their `aws` CLI configured first with like their root credentials, then create the policy, group, and user from CLI, then override `aws configure` using that user's credentials for `jets`?

It made a little more intuitive sense to me to sequence things by doing the IAM work in AWS console as root, then configure the CLI using the new IAM credentials. But I'm pretty inexperienced with this, hoping to learn more!

Reply:
Think it’s good to have console instructions also. Prefer it on a separate page and the pages cross link to each other.